Documentation/Signoffs/Motor System.md

```diff
@@ -0,0 +1,186 @@
+
+^ Source: https://learn.adafruit.com/all-about-stepper-motors  
+
+* **The motor will be protected at the nameplate current rating (6A).**  
+This is because a stepper motor divides a full rotation into a certain amount of equal steps. A stepper motor can do this by having groups of coils called "phases." When a "phase" is energized, the stepper motor will rotate by one step. The motor will only move on to the next step if the next "phase" group is energized. 
+
+^ Source: https://learn.adafruit.com/all-about-stepper-motors  
+240.4(B) is the standard for overcurrent devices rated 800 amperes or less, which allows the use of a higher rated overcurrent device as long as the conditions are met. All of these conditions have been cleared.  
+ * The conductors being protected are not part of a branch circuit supplying more than one receptacle for cord-and-plug-connected portable loads.  
+ * The ampacity of the conductors does not correspond with the standard ampere rating of a fuse or a circuit breaker without overload trip adjustments above its rating (but that shall be permitted to have other trip or rating adjustments).  
+ * The next higher standard rating selected does not exceed 800 amperes.  
+  
+^ Source: https://up.codes/viewer/illinois/nfpa-70-2020/chapter/2/wiring-and-protection#240.4_(B)  
+
+## Schematic
+
+**Motor**  
+![Motor](https://github.com/DillonSW/Capstone_Team_5/blob/main/images/MotorSchematic.jpg)
+
+This is the schematic for the stepper motor our team is looking to use, the E Series Nema 34 Stepper Motor. This motor has a holding torque of 12 Nm (or 106.29 lb-in). The calculations that led us to this decision is in Analysis. If we are able to go with this motor, we will need a driver. There is a driver built specifically for the Nema 34: The HSS86 Hybrid Step Servo.  
+^ Source: https://www.omc-stepperonline.com/download/34HE59-6004S.pdf  
+
+**Driver**  
+<img src="https://github.com/DillonSW/Capstone_Team_5/blob/main/images/DriverSchematic.jpg" width=50% height=50%>  
+
+This is the schematic for the frame of the servo, mainly used for implementation into a machine. All measurements are in mm.  
+^ Source: https://www.jbcnc.se/images/datasheets/HSS86.pdf  
+
+<img src="https://github.com/DillonSW/Capstone_Team_5/blob/main/images/DriverWiring.jpg" width=50% height=50%>  
+This is the wiring diagram for the driver and how it connects to the motor and step encoding. The resistors shown can vary, but because the signal control voltage will be +12V, the input port will need to be connected to a 1KΩ-2KΩ resistor.  
+
+## Analysis
+
+As stated previously, the motor must be able to output enough torque to rotate roughly 75 lbs. Our current machine design is a hollow cylinder, with the outer diameter being 3' and the inner, hollow diameter being roughly 1'4"  
+
+T=W(R1^2+R2^2)/2∗∆N/308t
+
+The equation above is for calculating the torque required to move a certain weight in a hollow cylinder.  
```

Comment:
You will need to explain this further. Just because a website says that it will work, doesn't mean that it will. Deeper analysis is needed.

I'm not saying this is wrong. I'm saying you seem to be taking the validity of this formula on blind faith.